hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancer.java

```diff
@@ -200,6 +228,8 @@ private boolean initializeIteration() {
     // find over and under utilized nodes
     for (DatanodeUsageInfo datanodeUsageInfo : datanodeUsageInfos) {
       double utilization = calculateUtilization(datanodeUsageInfo);
+      LOG.info("Utilization for node {} is {}",
+          datanodeUsageInfo.getDatanodeDetails().getUuidString(), utilization);
       if (utilization > upperLimit) {
         overUtilizedNodes.add(datanodeUsageInfo);
         countDatanodesToBalance += 1;
```

Comment:
remove `countDatanodesToBalance` , `overLoadedBytes`, `underLoadedBytes` ， no use here

Reply:
can you handle this? by the way, `canSizeLeaveSource`, `getClusterAvgUtilisation` seem not used now, we can remove them